.travis.yml

```diff
@@ -57,12 +57,7 @@ script:
       cd docs;
       make html;
       cd ..;
-      doctr deploy --deploy-repo xonsh/xonsh-docs dev;
```

Comment:
You need to revert all of the changes made to `.travis.yml` -- if these get merged into master it will break out docs deployment